api/all.yaml

```diff
@@ -1398,7 +1401,114 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ExternalLocationInfo"
-
+  /delta/preview/commits:
+    get:
+      operationId: getCommits
+      tags:
+        - CoordinatedCommits
+      summary: |
+        List unbackfilled Delta table commits.
+        WARNING: This API is experimental and may change in future versions.
+      description: |
+        List all the unbackfilled delta commits that are currently being tracked by the UC coordinator. 
+        If no commits are being tracked in the specific version range (from start_version to an optional end_version), 
+        it will return an empty list.
+        WARNING: This API is experimental and may change in future versions.
+      parameters:
+        - name: table_id
+          in: query
+          description: The ID of the table to get the commits for. This ID uniquely identifies a table.
+          required: true
+          schema:
+            type: string
+        - name: table_uri
+          in: query
+          description: |
+            The URI of the storage location of the table. If the table_id exists but the table_uri is 
+            different from the one previously registered (e.g., if the client moved the table), the request will fail.
+            Example: s3://bucket-name/tables/some-table-id
+          required: true
+          schema:
+            type: string
+        - name: start_version
+          in: query
+          description: |
+            The start version from which to retrieve commits (inclusive). This along with the
+            optional end_version specifies the range of commit versions that this request wants.
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: end_version
+          in: query
+          description: |
+            The end version upto which to retrieve commits (inclusive).
+            If not set, the latest version will be used as the end version. This does not affect the
+            latest_table_version in the response.
+            If num of commits that meet this criteria is > 50, the response will be paginated to the
+            first 50 commits. Call can send request again with a larger start_version to get the 
+            remaining commits.
```

Comment:
50 is an implementation choice the server can make based on the backend database requirements (how much store. from the catalog spec it perfectly correct to have the threshold be 1. please make that clear. 

in the implementation, this needs to be a server property